.github/workflows/aptos-node-release.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
           aptos_node_cargo_toml: aptos-node/Cargo.toml
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@671dc9c9e0c2d73f07fa45a3eb0220e1622f0c5f # pin@v4
+        uses: peter-evans/create-pull-request@a4f52f8033a6168103c2538976c07b467e8163bc # pin@v6.0.1
```

Comment:
Can't we just pin `@v6` instead ? Pinning to commit IDs gets old soon.

Reply:
We don't update this dep too often, and since it's not hosted on an official org, I'd usually choose to be paranoid and pin by commit hash

Reply:
Ok.